openpgp/ocb/ocb.go

```diff
@@ -113,17 +113,16 @@ func (o *ocb) crypt(instruction string, nonce, adata, text []byte) []byte {
 	X := make([]byte, len(text))
 	copy(X, text)
 	blockSize := o.block.BlockSize()
-	tagSize := o.TagSize()
 
 
 	//
 	// Nonce-dependent and per-encryption variables
 	//
 	// From RFC 7253:
 	// Nonce = num2str(TAGLEN mod 128, 7) || zeros(120 - bitlen(N)) || 1 || N
-	paddedNonce := append(make([]byte, tagSize-1-len(nonce)), 1)
+	paddedNonce := append(make([]byte, blockSize-1-len(nonce)), 1)
```

Comment:
@twiss, In case anyone uses short tagSizes, this would have generated an awful out of index error.

Reply:
(The typo was the use of `tagSize` instead of `blockSize` in previous line)